lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +194,70 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (this.typingTimeout) clearTimeout(this.typingTimeout);
```

Comment:
use curly braces, even if it is single line if.

Reply:
On it.